src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```diff
@@ -3937,8 +3937,8 @@ compiler.err.primitive.class.does.not.support=\
 compiler.err.value.class.may.not.extend=\
     inappropriate super class declaration for a value class
 
-compiler.err.primitive.class.instance.field.expected.here=\
-    withfield operator requires an instance field of a primitive class here
+compiler.err.primitive.or.value.class.instance.field.expected.here=\
+    withfield operator requires an instance field of a primitive or a value class here
 
```

Comment:
This message is not in alignment with rest of diagnostics that ony speak of value classes. I suggest we amend this to 
compiler.err.value.class.instance.field.expected.here. I think it is very visible and open that every primitive class is a value class so it is not necessary to call out primitive or value class